OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -96,28 +99,28 @@ void TracerThread::Run(
     perf_event_enable(fd_to_ring_buffer.first);
   }
 
-  // Record and periodically print basic statistics on the number events.
-  constexpr uint64_t EVENT_COUNT_WINDOW_S = 5;
-  uint64_t event_count_window_begin_ns = 0;
-  uint64_t sched_switch_count = 0;
-  uint64_t sample_count = 0;
-  uint64_t uprobes_count = 0;
+  event_count_window_begin_ns_ = 0;
```

Comment:
Below you use UpdateStats(), for symmetry I would suggest a ResetStats() here. Should also make it easier to add / update stats when UpdateStats and ResetStats are defined next to each other.